openpype/pipeline/publish/lib.py

```diff
@@ -632,3 +632,49 @@ def get_instance_staging_dir(instance):
         instance.data["stagingDir"] = staging_dir
 
     return staging_dir
+
+
+def get_publish_repre_path(instance, repre, only_published):
```

Comment:
I would make the `only_published` optional.
```suggestion
def get_publish_repre_path(instance, repre, only_published=False):
```

Reply:
Yeah good call, done!